docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+services:
+
+  app:
+    image: ghcr.io/radu-c-martin/maybe:latest
+    ports:
+      - 3000:3000
+    restart: unless-stopped
+    env_file:
+      - .env
+    environment:
+      DB_HOST: "postgres"
+      HOSTING_PLATFORM: "localhost"
+      DISABLE_SSL: "true"
+    depends_on:
+      postgres:
+        condition: service_healthy
+
+  postgres:
+    image: postgres:16
+    restart: unless-stopped
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER:?}
+      POSTGRES_DB: ${POSTGRES_DB:-postgres}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:?}
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U $$POSTGRES_USER" ]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+volumes:
+  postgres-data:
```

Comment:
unless you're expecting devs/contributors to use this file to run it locally, i'd rename it to something like `docker-compose.example.yml`

Reply:
That's a good point—we will _not_ be supporting a dev-optimized compose file since that is already done in `.devcontainer`.  It should be made clear that this should not be used for development.

We should also throw a noticeable comment at the top of the file with some links to the documentation on how to get it running for self-hosting